src/main/kotlin/io/titandata/titan/providers/local/Run.kt

```diff
@@ -61,14 +71,15 @@ class Run (
         argList.add("--name")
         argList.add(containerName)
         argList.addAll(argumentEdit)
-        val repoDigest = imageInfo.getJSONArray("RepoDigests")[0] as String
+        val imageSHA = imageInfo.getJSONObject("ContainerConfig").getString("Image")
```

Comment:
I think this should be Config, not ContainerConfig. The latter is more about the history, and apparently can be blacnk:

https://stackoverflow.com/questions/36216220/what-is-different-of-config-and-containerconfig-of-docker-inspect

Reply:
Updating